iodata/utils.py

```diff
@@ -37,7 +37,10 @@
 electronvolt: float = 1 / spc.value(u'hartree-electron volt relationship')
 # atomic mass unit (not atomic unit of mass!)
 amu: float = 1e-3 / (spc.value(u'electron mass') * spc.value(u'Avogadro constant'))
-
+# Avogadro constant
+avogadro = spc.value('Avogadro constant')
+# calorie
+calorie = spc.calorie
```

Comment:
This would be good if IOData would use SI units internally. However, we use atomic units instead. Could you define the following unit, instead of having avogadro and calorie separately?

```python
kcalmol = 1e3 * spc.calorie / spc.value('Avogadro constant') / spc.value('Hartree energy')
```